src/middleware/articleMiddleware.js

```diff
@@ -0,0 +1,38 @@
+import ArticleRateHelper from '../helpers/articles/rateArticleHelper';
+
+/**
+ * @exports ArticleMiddleware
+ * @class ArticleMiddleware
+ * @description Article Middleware
+ * */
+class ArticleMiddleware {
+  /**
+     * Validate article and rate
+     * @param {string} req - a request
+     * @param {string} res - a response
+     * @param {string} next - next function for passing to the next function
+     * @return {string} Returns error or proceed the execution
+     * @static
+     */
+  static async checkRatedArticle(req, res, next) {
+    const errorMessage = [];
+    const validateArticle = await ArticleRateHelper.validateArticleRated(
+      req.params.id, req.userData.id
+    );
+    if (validateArticle !== true) {
```

Comment:
It's more fun using shorthand coding techniques. Instead of `validateArticles !== true`, better to use `!validateArticles`. Check out this link from @Jkadhuwa yesterday
https://www.sitepoint.com/shorthand-javascript-techniques/

Reply:
Yes @Malaba6 shorthands are very good, however, they can not be applied to all scenarios eg like this one, the shorthand check will pass only as long as likeJavascript is a truthy value, for more information read this https://developer.mozilla.org/en-US/docs/Glossary/Truthy